.github/workflows/test.yaml

```diff
@@ -5,7 +5,6 @@ on:
     branches:
       - main
   pull_request:
-  workflow_dispatch:
```

Comment:
forgot to ask: any particular reason you wanted to remove this event in this repo? I'm not necessarily opposed to removing it, as I think it's not super necessary (?), but we might want to be consistent across test wfs! (See your corresponding PR for the f-API https://github.com/neurobagel/federation-api/pull/136/files)

Reply:
To me that's a holdover from WF debugging. When would we want to trigger tests manually and haven't run them already? But agree, let's do this consistently across repos